### ISSUE TITLE
Vertica Support

### ISSUE THREAD

**Issue by ghayes** [ankane/blazer#55]
We currently use PeriscopeData so I love the direction this project is taking. The only thing holding me back from introducing Blazer to a broader group is that we use Vertica instead of Redshift as our data warehouse.  Are there any plans to add support for Vertica?

**Comment by ankane** [comment]
Hi Greg, Blazer should work with any DB that has an ActiveRecord adapter.

You could try this: https://github.com/evrone/activerecord-vertica-adapter

**Comment by ankane** [comment]
Cleaning up issues.  Let me know if you have any luck.

**Comment by ghayes** [comment]
Hi Andrew,  I have made a little progress with Vertica.  The link you provided hasn't been updated in a few years and so was dependent on an older version of ActiveRecord (3.X I believe).  I did find another project called "vertica_rails_adapter".  I had to make a couple of changes but did manage to get a connection.  You have a query to retrieve table and column info.  In Vertica, the "columns" view is accessed via the "v_catalog" schema instead of the "information_schema".  Once I had that figured out I was able to attempt a simple "count(*)" query against one of my tables, but ended up with a Invalid Data Type error.  So that will be the next issue I tackle.
